_i18n/en/documentation/general/f-droid.md

```diff
@@ -2,17 +2,39 @@ Did you see that there's a new AntennaPod release and wonder:
+### The steps for releasing a new version on F-Droid
+1. We release beta versions, which are _only_ available through Google Play due to [technical limitations](/documentation/general/beta#f-droid) in F-Droid.
+2. We do a slow gradual roll-out to Google Play users. This makes sure that there are no critical bugs and can take several weeks.
+3. We create a [release on GitHub](https://github.com/AntennaPod/AntennaPod/releases). We do this only after the roll-out on Google Play is complete.
+4. F-Droid systems check the AntennaPod repository and detect a new release. The update checker server runs daily (overnight in Europe).
+5. F-Droid systems build the app. They do this to make sure the app you download matches the source code that we publishes. Because there are many apps to build, the F-Droid server takes 1 to 2 days to build the app.
+6. An F-Droid contributor signs the build cryptographically. To improve security, the contributor needs to take the app and physically walk it to a dedicated computer which is not connected to the internet. When all apps are signed, the contributor uploads them to the F-Droid systems. This usually takes 1 to 2 days.
+7. F-Droid systems process the new apps, adding the new AntennaPod release in their index.
 
-### What you can do if it takes longer than expected
-As you can see it can easily take a week before a new AntennaPod release is available on F-Droid. If an update is still not available more than a week after its release, feel free to let us know by creating a post on our [forum](https://forum.antennapod.org/), and we'll investigate it.
+At this point we consider it 'released on F-Droid'. But remember that your F-Droid client (the 'app store') also needs to refresh its local copy of the index. Only after that you'll get promted about the latest AntennaPod release.
 
-You can get an idea of the progress by checking [if F-Droid's releases list was updated](https://gitlab.com/fdroid/fdroiddata/-/commits/master?search=Update+known+apks) since the new AntennaPod release and by checking on [F-Droid Monitor](https://monitor.f-droid.org/builds/build) if the new release was built yet.
+### So, what's the ETA?
+As you can see, there are a lot of steps which each take a varying amount of time. We usually take well over a month between releasing a beta version and tagging the release on GitHub. After the release is tagged, F-Droid usually takes four to six days to process it.
+
+### What you can do if you're eagerly waiting
+As the F-Droid team [notes](https://gitlab.com/fdroid/wiki/-/wikis/FAQ#how-long-does-it-take-for-my-app-to-show-up-on-website-and-client): "don't panic before 7 days have passed, please."
+
+* Did those 7 days pass since we tagged the release on GitHub? Feel free to let us know by creating a post on our [forum](https://forum.antennapod.org/). We'll take a look.
+* Did we tag the release 7 days ago or less? Then please just wait. Or investigate where we are in the process.
```

Comment:
- Did we not tag the release yet? The update is not ready for F-Droid yet

Maybe link to the releases here also?